@FISHDATA/plotDist2centerline.m

```diff
@@ -1,7 +1,24 @@
-function plotDist2centerline(obj)
+%  Filename: plotDist2centerline.m (method of FISHDATA class)
+%  Abstract:
+%        Plot distance to centerline of the arena for a FISHDATA.
+%
+%   SYNTAX:
+%       1. obj.plotDist2centerline(phase)
+%       2. plotDist2centerline(obj,phase)
+%      
+% - plotDistance2centerline:
+%      - FISHDATA method
+%      - arg1: the entire process / test-phase only / baseline-phase / training-phase
+%      - arg2: pixelSize
+%      - arg3: w/o extinction point (blue triangle)
```

Comment:
it would be better to have more input arguments so that our users would have more freedom to decide which labels to be shown on the figure.